### PR TITLE
Remove positive lookbehind regex as it is only supported in chrome.

### PR DESCRIPTION
**Pull request by jkeck**

### CODE REVIEW

File: src/lib/AnnotationResource.js [hunk 1]
```diff
@@ -37,9 +37,9 @@ export default class AnnotationResource {
     const { on } = this.resource;
     switch (typeof on) {
       case 'string':
-        return on.match(/(?<=xywh=).*$/)[0].split(',');
+        return on.match(/#xywh=(.*)$/)[1].split(',');
       case 'object':
-        return on.selector.value.match(/(?<=xywh=).*$/)[0].split(',');
+        return on.selector.value.match(/#xywh=(.*)$/)[1].split(',');
```
Comment (jkeck) [inline src/lib/AnnotationResource.js]: The tests pass, but not 100% sure if there are other cases I'm missing that the lookbehind was getting us.

Comment (cbeer) [reply src/lib/AnnotationResource.js]: I don't think we need the `#`; at least the previous implementation didn't care.

Comment (jkeck) [reply src/lib/AnnotationResource.js]: Yeah, looks like while it is recommended, is not required.

**Pull request merged**